src/pocketmine/utils/TextFormat.php

```diff
@@ -64,6 +64,17 @@
 	public static function tokenize(string $string) : array{
 		return preg_split("/(" . TextFormat::ESCAPE . "[0123456789abcdefklmnor])/", $string, -1, PREG_SPLIT_NO_EMPTY | PREG_SPLIT_DELIM_CAPTURE);
 	}
+	
+	/**
```

Comment:
formatting

Reply:
formatting, yeah. Doesnt get what do you mean :\

Reply:
Wrong indentation, looks like an extra tab.

Reply:
how to fix?

Reply:
Remove this
![screenshot_20171220-162009](https://user-images.githubusercontent.com/15074389/34206761-540ac340-e5a1-11e7-9102-b0374390dc77.jpg)